### ISSUE TITLE
Tied notes are articulated

### ISSUE THREAD

**Issue by VictorEijkhout** [musescore/MuseScore#19420]
### Issue type

General playback bug

### Bug description

This is a document from the experimental pdf convertor. A number of tied notes lost their ties, so I introduced them by hand. The tie in measure 6 of the attached score is not reflected in playback: the second note is articulated.

### Steps to reproduce

1. No idea how to reproduce it. You'll have to look into the innards of the score.

2. (Can you tell github that `.mscz` is a legitimate extension for files to upload here?)
[bug.zip](https://github.com/musescore/MuseScore/files/12641426/bug.zip)

### MuseScore Version

4.1.1

### Regression

I don't know

### Operating system

MacOs 13

**Comment by VictorEijkhout** [comment]
Attaching the output of the pdf convertor.
[scan.zip](https://github.com/musescore/MuseScore/files/12641431/scan.zip)

**Comment by MarcSabatella** [comment]
That's a slur, not a tie - so the playback is correct.

**Comment by VictorEijkhout** [comment]
Between identical pitches, what's the difference? Doesn't a slur meaning play bunch of notes without articulation after the first? So no tonguing for wind instruments or bow direction change for bowed strings?

Practically: there is no way to tie notes after the fact, I have to re-enter them?

**Comment by MarcSabatella** [comment]
You can certainly add ties after the fact - select the first note and issue the tie command (press "T" or click icon).

Skurs are not ties.  They are handled differently - different attachment points relative to notehead, different behavior when adding to chords, different rules for positioning with multiple voices, different behavior if you change the pitch of one of the two notes, etc.  Slurs are played differently by different instruments, but even on winds, two notes of the same pitched slurred can be played with a "breath accent" or light tongue.

Bottom line, don't use a slur when you mean a tie 0 they just aren't the same at all.

**Comment by zacjansheski** [comment]
Closing, please refer to comments above, thank you.
